pom.xml

```diff
@@ -64,21 +64,27 @@
         <cxf.version>3.1.4</cxf.version>
         <cxf.xpp3.bundle.version>1.1.4c_6</cxf.xpp3.bundle.version>
         <jts.bundle.version>1.12_1</jts.bundle.version>
-        <karaf.version>2.4.3</karaf.version>
+        <karaf.version>4.0.4</karaf.version>
         <lux.version>1.2</lux.version>
         <opengis.bundle.version>${org.geotools.version}_1</opengis.bundle.version>
         <org.geotools.bundle.version>${org.geotools.version}_1</org.geotools.bundle.version>
         <org.geotools.version>8.4</org.geotools.version>
         <org.slf4j.version>1.7.1</org.slf4j.version>
```

Comment:
Is that issue going to be fixed? It looks like that ticket has been around for over 3 years now. 

Of note though, I should have rephrased...karaf is bringing in a new version of slf4j, so it might be worthwhile to verify the compile versions in here match the runtime versions that karaf is bringing in. It looks like you've already done that for most of the items so it should be good. Just note that if there is an issue with the new version of slf4j...that's the version that will be used at runtime. 

On the same note, it would probably be good to git rid of the slf4j-ext dependency. I believe it is slowly being phased out, however with a big change like this happening might as well just get that done fully (and if that's what is causing the issue with cal10n-api, then it should be part of this karaf upgrade ticket).